contract/test/test-launchIt.js

```diff
@@ -28,9 +28,6 @@ const bundleRoots = {
   contractStarter: nodeRequire.resolve('../src/contractStarter.js'),
 };
 
-/** @type {import('ava').TestFn<Awaited<ReturnType<makeBundleCacheContext>>>} */
-const test = anyTest;
-
```

Comment:
I don't see how this change is relevant. What happens if you revert it? (all 3 times)